mget.go

```diff
@@ -1,15 +1,17 @@
 package opensearchtools
 
 import (
-	"bytes"
 	"context"
 	"encoding/json"
 	"net/http"
-
-	"github.com/opensearch-project/opensearch-go/v2"
-	"github.com/opensearch-project/opensearch-go/v2/opensearchapi"
 )
 
+// MGet defines a method which knows how to make an OpenSearch multiple document get.
```

Comment:
I would change this comment, as it doesn't quite line up with OpenSearch verbiage.
```
// MGet defines a method which knows how to make an OpenSearch [Multi-get] request.
// It should be implemented by a version-specific executor.
//
// [Multi-get]: https://opensearch.org/docs/latest/api-reference/document-apis/multi-get/
```

Reply:
Thanks - done